env0/data_credentials.go

```diff
@@ -2,13 +2,21 @@ package env0
 
 import (
 	"context"
-	"strings"
 
 	"github.com/env0/terraform-provider-env0/client"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+var credentialsTypeToPrefixList map[string][]string = map[string][]string{
```

Comment:
Based on the feedback I received in Slack. This is the correct mapping.
If I got anything wrong, please let me know.